adafruit_circuitplayground/express.py

```diff
@@ -88,6 +88,39 @@ def acceleration(self):
         """
         return self._lis3dh.acceleration
 
+    @property
```

Comment:
I don't think this should be a property because the implementation sleeps. So, remove property here and below return `self._lis3dh.shake()`. Right now you are actually returning the underlying function as the value of the property and then calling it. After the changes you'll return the value itself.